homeassistant/components/browan/sensor.py

```diff
@@ -0,0 +1,195 @@
+"""Support for MQTT sensors."""
+from __future__ import annotations
+
+from collections.abc import Awaitable, Callable
+from dataclasses import dataclass
+import json
+import logging
+
+from pyliblorawan.models import Uplink
+from pyliblorawan.network_servers.ttn import TTN
+
+from homeassistant.components.mqtt.models import ReceiveMessage
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    ATTR_BATTERY_LEVEL,
+    ATTR_TEMPERATURE,
+    UnitOfElectricPotential,
+    UnitOfTemperature,
+)
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+)
+
+from . import devices
+from .const import DOMAIN, MANUFACTURER, TTN_TOPIC
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@dataclass
+class BrowanSensorDescriptionMixin:
+    """Mixin for Browan sensor."""
+
+
+@dataclass
+class BrowanSensorEntityDescription(
+    SensorEntityDescription, BrowanSensorDescriptionMixin
+):
+    """Class describing Browan sensor entities."""
+
+    name: str | None = None
```

Comment:
are we using that?

Reply:
Yes, while it is not necessary for the code to execute it is helpful for Mypy